shared/i18n/locales/zh_CN/translation.json

```diff
@@ -1017,8 +1017,12 @@
   "Configure a Matomo installation to send views and analytics from the workspace to your own Matomo instance.": "配置一个 Matomo 安装，将视图和分析从工作区发送到你的 Matomo 实例。",
   "Instance URL": "实例 URL",
   "The URL of your Matomo instance. If you are using Matomo Cloud it will end in matomo.cloud/": "你的 Matomo 实例 URL。如果你使用 Matomo 云，它将在 matomo.cloud/ 中结束。",
+  "The URL of your Umami instance. If you are using Umami Cloud it will end in https://cloud.umami.is/": "你的 Umami 实例 URL。如果你使用 Umami Cloud，请配置 https://cloud.umami.is/。",
```

Comment:
Remove these changes as the file is auto-generated from [CrowdIn](https://translate.getoutline.com/)